src/components/SearchBar/SearchBar.module.css

```diff
@@ -6,6 +6,7 @@
   transform: translateX(-2rem);
   margin-top: 0.3rem;
   position: absolute;
+  cursor: pointer;
```

Comment:
There's an accessibility discussion about this on [the interwebs](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b), although I believe you just need enough people that feel like pointer cursors on buttons are good